modules/unsupported/arcgis-rest/pom.xml

```diff
@@ -111,16 +103,21 @@
         </dependency>
         <dependency>
             <groupId>org.powermock</groupId>
-            <artifactId>powermock-api-mockito</artifactId>
+            <artifactId>powermock-api-mockito2</artifactId>
             <version>${powermock.version}</version>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.geotools</groupId>
             <artifactId>gt-cql</artifactId>
-            <version>23-SNAPSHOT</version>
+            <version>25-SNAPSHOT</version>
```

Comment:
You should be able to use ${project.version}  here AFAIK

Reply:
Done.